Public/Src/Engine/ViewModel/BuildViewModel.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using BuildXL.Pips;
+using BuildXL.Utilities;
+
+namespace BuildXL.ViewModel
+{
+    /// <summary>
+    /// Provides access to internals of the build engine used to provide the user experience.
+    /// This can be for the fancy console to provide progress or to emit azure devops markers.

```

Comment:
>fancy console to provide progress or to emit azure devops markers. [](start = 28, length = 66)

Not the content of this PR but this comment reminds me that /ado mode should disable /fancyconsole

Reply:
It already does :)